### PR TITLE
homepage done

### PR DESCRIPTION
**Pull request by cchchoi1986**
whatup

### CODE REVIEW

File: app/assets/javascripts/angular/controllers/home_controller.js.erb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+app.controller('HomeCtrl', ['$scope', '$http', '$routeParams', function($scope, $http, $routeParams){
+  $scope.mainNavBar = "<%= asset_path('navBarPartial.html') %>";
+  $scope.mainFooter = "<%= asset_path('footerPartial.html') %>";
+  $scope.course = "<img class='folder-courses' height='72px' width='72px'>";
```
Comment (h8rry) [inline app/assets/javascripts/angular/controllers/home_controller.js.erb:4]: Why do you need to define HTML and CSS here?

File: app/assets/stylesheets/recipe_finder.css.scss [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  height: 580px;
+  border-radius: 10px;
+  padding: 20px;
+}
+
+.recipe-profile {
+  background: #666;
+}
+
+@media (max-width: )
```
Comment (h8rry) [inline app/assets/stylesheets/recipe_finder.css.scss]: What is this for? It's breaking the site

File: app/assets/stylesheets/static_pages.css.scss [hunk 3]
```diff
@@ -1,3 +1,275 @@
-// Place all the styles related to the static_pages controller here.
-// They will automatically be included in application.css.
-// You can use Sass (SCSS) here: http://sass-lang.com/
+$color1: #FFF888;
```
Comment (h8rry) [inline app/assets/stylesheets/static_pages.css.scss]: - A more descriptive name would be preferred
- Same applies to the rest of the CSS doc

File: app/assets/stylesheets/static_pages.css.scss [hunk 4]
```diff
@@ -1,3 +1,275 @@
...
+$font3: 'Bitter', serif;
+// $font3: 'Lobster', cursive;
+// $font3: 'Special Elite', cursive;
+$font: 'Satisfy', cursive;
+// $font3: 'Damion', cursive;
+$font2: 'Architects Daughter', cursive;
+// $font3: 'Cookie', cursive;
+
+.body {
+  background: $color5;
```
Comment (h8rry) [inline app/assets/stylesheets/static_pages.css.scss]: Very hard to understand what $color5 is...

**Comment by h8rry** [comment]
Please take a look at my comments. The rest looks good. Did you push to heroku to see if it works on production?

File: app/assets/javascripts/angular/controllers/home_controller.js.erb (hunk 1 above)
Comment (cchchoi1986) [inline app/assets/javascripts/angular/controllers/home_controller.js.erb:4]: the img was just a test, the others are for my nav and footer bars

File: app/assets/stylesheets/recipe_finder.css.scss (hunk 2 above)
Comment (cchchoi1986) [inline app/assets/stylesheets/recipe_finder.css.scss]: this is the next page for recipes search

File: app/assets/stylesheets/static_pages.css.scss (hunk 3 above)
Comment (cchchoi1986) [inline app/assets/stylesheets/static_pages.css.scss]: ok I'll change to be more descriptive

File: app/assets/stylesheets/static_pages.css.scss (hunk 4 above)
Comment (cchchoi1986) [inline app/assets/stylesheets/static_pages.css.scss]: ditto

**Pull request merged**